src/detector/detector.py

```diff
@@ -15,6 +15,9 @@
     StartCaptureRequest, \
     StopCaptureRequest
 from src.detector.fileio import DetectorConfiguration
+from src.calibrator import Calibrator
+from src.calibrator.fileio import CalibratorConfiguration
+from src.calibrator.api import *
```

Comment:
Generally it's best to spell out exactly what you're importing; It'll be easier to find where those classes are used.